SAS/ttest.qmd

```diff
@@ -0,0 +1,78 @@
+---
+title: "Students t-test"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+### **Independent t-test in SAS**
+
+The null hypothesis of the Independent Samples t Test is, the means for the two populations are equal.
+
+In SAS the following code was used to test the mean comparison (mean of Weight Gain) of two independent treatment groups (Treatment and Placebo).
+
+For this example, we're testing the significant difference in mean of Weight gain (*WtGain*) between treatment and placebo (*trt_grp*) using PROC TTEST procedure in SAS.
+
+```{r}
+#| eval: false 
+  proc ttest data=d1; 
+     class trt_grp; 
+     var WtGain; 
+  run; 
+```
+
+Output:
+
+```         
+             Figure 1: Test results for independent t-test using PROC TTEST in SAS
+```
+
+```{r, echo=FALSE, fig.align='center', out.width="50%"}
+knitr::include_graphics("../images/ttest/test1.png")
+```
+
+Here the t value is --0.70, degrees of freedom is 30 and P value is 0.4912 which is greater than 0.05, so we accept the null hypothesis that there is a significant difference between the means of treatment groups. Also the mean difference is --7.9375 and the 95% CI for the mean difference is [--31.1984, 15.3234].
+
+Note: Before entering straight into the t-test we need to check whether the assumptions (like the equality of variance, the observations should be independent, observations should be normally distributed) are met or not. If normality is not satisfied, we need to go with the non-parametric test of independent t test.
```

Comment:
Please change "we need to go with the non-parametric test of independent t test."   
to:  "we may consider using a suitable non-parametric test."